options/options.cc

```diff
@@ -477,15 +477,29 @@ ColumnFamilyOptions* ColumnFamilyOptions::OptimizeForSmallDb() {
   max_bytes_for_level_base = 10 * 1048576;
   soft_pending_compaction_bytes_limit = 256 * 1048576;
   hard_pending_compaction_bytes_limit = 1073741824ul;
+
+  // 16MB block cache
+  std::shared_ptr<Cache> cache = NewLRUCache(16 << 20);
+  BlockBasedTableOptions table_options;
+  table_options.block_cache = cache;
+  table_options.cache_index_and_filter_blocks = true;
+  // Two level iterator to avoid LRU cache imbalance
+  table_options.index_type =
+      BlockBasedTableOptions::IndexType::kTwoLevelIndexSearch;
+  table_factory.reset(new BlockBasedTableFactory(table_options));
+
+  // Cost memtable to block cache too.
+  std::shared_ptr<rocksdb::WriteBufferManager> write_buffer_manager =
```

Comment:
Wouldn't these changes impact the existing users of this API? 
Though these changes are good for users who create a new DB, I am trying to think though all the ways the existing users of this API would be impacted.

Reply:
After thinking through, I think it might not be a concern.